jscomp/main/dune

```diff
@@ -26,3 +26,10 @@
  (package melange)
  (modules ounit_tests_main)
  (libraries ounit2 ounit_tests))
+
+(executable
+ (name jsoo_main)
+ (modules jsoo_main jsoo_common)
+ ; Need to use melange_ppx, not melange_ppx_lib, so Driver.V2.register_transformation runs
```

Comment:
`melange.ppx-lib` is not a thing anymore after #583, the comment can be updated.

Reply:
https://github.com/melange-re/melange/pull/538/commits/c4ad80d59c355c0478dc3787ad404386d212c11a